src/main/resources/db/migration/V1_9__create_audit_table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS audit (
+  id                    INTEGER                    NOT NULL
+, case_id               INTEGER                    NOT NULL
+, event_id              INTEGER                    NOT NULL
+, moj_usr_id            INTEGER                    NOT NULL
```

Comment:
Should this column be `user_id` as it looks different from the others?

Table appears to be slightly different here than what's specified in the Data Model section of https://tools.hmcts.net/confluence/display/DMP/Case+audit+service+LLD
?

Reply:
ACCEPT.